cmd/skaffold/app/cmd/fix.go

```diff
@@ -121,8 +121,16 @@ func fix(out io.Writer, configFile, outFile string, toVersion string) error {
 	if outFile != "" {
 		var mvErr error
 		if overwrite {
+			if err != nil {
+				return fmt.Errorf("reading config file: %w", err)
+			}
+			oldCfg, readErr := os.ReadFile(configFile)
+			if readErr != nil {
+				fmt.Errorf("reading config file: %w", readErr)
+			}
 			mvFile := fmt.Sprintf("%s.v2", outFile)
```

Comment:
nit, consider changing prefix now that the file is not being moved

Reply:
Done.